architect/systems/optical/spectrometers.py

```diff
@@ -303,14 +303,20 @@ def get_pointing_accuracy_constraint(
 
         return constraint_angle
 
-    def get_ground_target_error(self, orbital_altitude, skew_angle):
+    def get_ground_target_error(self, orbital_altitude, desired_pointing_angle, skew_angle):
```

Comment:
```suggestion
    def get_ground_target_error(self, orbital_altitude, skew_angle, pointing_accuracy: unit.deg = 0):
```
See comment in https://www.notion.so/utat-ss/Ground-Target-Error-When-Not-Pointing-Directly-at-Nadir-22b3069f4a0344b08339e5004f90438b for renaming variable names